public/app/features/alerting/unified/components/rules/AlertInstancesTable.tsx

```diff
@@ -34,30 +35,18 @@ export const AlertInstancesTable: FC<Props> = ({ instances }) => {
       isExpandable={true}
       items={items}
       renderExpandedContent={({ data }) => <AlertInstanceDetails instance={data} />}
+      pagination={pagination}
+      footerRow={footerRow}
     />
   );
 };
 
-export const getStyles = (theme: GrafanaTheme2) => ({
-  colExpand: css`
```

Comment:
It looks like these styles were never used? 😅

Reply:
Yes, what's interesting is that mine is not used either 😇 Eslint has been fooled due to `export` and doesn't complain that the method is not used. I'll fix